examples/jupyter/Quickstart.ipynb

```diff
@@ -406,8 +408,41 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Step 3: Fitting\n",
-    "\n",
+    "## Step 3: Fitting"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. note::\n",
+    "    We load the relevant data first, so that we can run this step\n",
+    "    independently from the others. These commands can be considered a summary\n",
+    "    of the previous steps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reset -f\n",
+    "import pycompwa.ui as pwa\n",
+    "particle_list = pwa.read_particles('model.xml')\n",
+    "kinematics = pwa.create_helicity_kinematics('model.xml', particle_list)\n",
+    "kinematics.create_all_subsystems()\n",
+    "data_sample = pwa.read_root_data(input_file='generated_data.root')\n",
+    "phsp_sample = pwa.read_root_data(input_file='generated_phsp.root')\n",
+    "intensity_builder = pwa.IntensityBuilderXML(\n",
+    "    'model.xml', particle_list, kinematics, phsp_sample)\n",
+    "intensity = intensity_builder.create_intensity()"
+   ]
+  },
```

Comment:
Does this mean the whole code is executed again if someone is going through the whole example?

Reply:
Yes but is a matter of seconds

Reply:
Hmm, so how does this help a user with his quickstart? To me this quickstart notebook should just show how to usage and process from the beginning to the end looks like.

Reply:
Yes that is true. I even considered splitting up the notebook and then having them appear on different pages on the [examples webpage](https://compwa.github.io/examples.html). Then you can cycle through them (like a button "next step" in the bottom), so you can cycle through them like a tutorial. In the backend you then also have a different notebooks for each step, because in practice, if set up your own analysis, you will want to split up those scripts anyway.

Reply:
I really see more harm then good in keep repeating the previous tasks. Especially if you have to run the whole code once from the beginning anyway (to create the model.xml and data). So you do not save anything if you want to start at the fitting section (unless you ran this stuff before and still have data files).
If these quickstart is split up it would make more sense. But I would still add some remarks about the requirements (so what has to be run before this actually can run)